### PR TITLE
Revert to 14 point sides

### PR DESCRIPTION
**Pull request by vkantsev**
The extra `corner` values were confusing. Revert to the original `board_side == playershift == 14`.

This would also allow us to reuse `console_view.draw_board()`.

### CODE REVIEW

File: src/state.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def is_on_finish(self, piece: Piece) -> bool:
         return self.path_zone_length < piece.position < self.end_progress
 
     def is_on_target(self, piece: Piece) -> bool:
-        return piece.position > (self.path_zone_length + self.finish_zone_length)
+        return self.end_progress <= piece.position <= self.end_progress + 3
```
Comment (mapto) [inline src/state.py:123]: Why is this changed? What does the + 3 mean?

Comment (vkantsev) [reply src/state.py:119]: 👋 If we consider that the `target` starts at the first position outside the _normal_ pathways  (i.e. `path_zone_length` + `finish_zone_length` + `1`) and that it would eventually host `4` pieces then we could effectively state that its `length` is equal to `4` or `end_progress` + `3`.
